index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="shortcut icon"
+      href="https://favicon-generator.org/favicon-generator/htdocs/favicons/2014-11-23/641ccf83998115288af7e77f866fb65c.ico"
+      type="image/x-icon"
+    />
+    <title>Apple</title>
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      
+      <div class="image0">
+        <nav>
+            <ul>
+              <li><a href="#"> <i class="fa fa-apple" aria-hidden="true"></i></a></li>
+              <li><a href="#"> Store </a></li>
+              <li><a href="#"> Mac </a></li>
+              <li><a href="#"> iPod</a></li>
+              <li><a href="#"> iPhone</a></li>
+              <li><a href="#"> iPad</a></li>
+              <li><a href="#"> iTunes</a></li>
+              <li><a href="#"> Support</a></li>
+              <li><input type="search" name="search" id="search" /></li>
+            </ul>
+        </nav>
+        <div class="text">
+            <h1 id="main-txt">What will your verse be ?</h1>
+            <br />
+            <h2 id="sub-txt">Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.</h2>
+            <h3 class="sub-heading">Watch the “Your Verse” film and explore the stories</h3>
+        </div>
+      </div>
+      <div class="image1"></div>
+      <div class="image2"></div>
+      <div class="image3"></div>
+      <div class="image4"></div>
+        
```

Comment:
these images should be links please wrap them inside a link tag or change the div into a link tag

Reply:
doing so, wrapping them inside a `<a>` or `<link>` tags make them disappear from the page